_studio/shared/src/mfx_vpp_vaapi.cpp

```diff
@@ -253,6 +256,16 @@ mfxStatus VAAPIVideoProcessing::Init(_mfxPlatformAccelerationService* pVADisplay
             return MFX_ERR_DEVICE_FAILED;
         }
 
+        std::vector<VAConfigAttrib> attrib(1);
```

Comment:
Why you don't do that when VP queries capabilities here:
https://github.com/Intel-Media-SDK/MediaSDK/blob/48f48caf166e5231cf2e2f60a97f0ecd9279a48b/_studio/shared/src/mfx_vpp_vaapi.cpp#L308

Reply:
Done.